testfiles/_SmallOffice_Dulles.idf

```diff
@@ -5057,9 +5057,9 @@
     SumOrAverage,            !- Aggregation Type for Variable or Meter 2
     Zone Windows Total Transmitted Solar Radiation Rate,  !- Variable or Meter 3 Name
```

Comment:
Missed some conversions here (AUD diffs were the hint something was up). There are also AUD diffs in 5ZoneEconomicsTariffAndLifeCycleCosts, 5ZoneTDV, and several other files.

@Myoldmopar When I run this file locally, I see err diffs. There's a new warning with this branch, but no err diffs on ci:
```
   ** Warning ** Processing Monthly Tabular Reports: Variable names not valid for this simulation
   **   ~~~   ** ...use Output:Diagnostics,DisplayExtraWarnings; to show more details on individual variables.
```
And, duh, just realized, that's because locally it ran a RunPeriod simulation, but CI didn't, so these monthly reports didn't process.

Reply:
Huh, weird.  Ok, I see that somehow I missed some of these IDF files.  Not sure how, unless these were modified during recent commits.  But it is strange that it didn't show up in the CI.  Anyway, I'm working my way through this again.  Should have another commit to fix these issues today.

Reply:
Ok, just made another commit to address this.  Should be all of them now.  Thanks!